services/brig/deb/opt/brig/templates/user/de/email/activation.html

```diff
@@ -48,7 +48,7 @@
 
           <p style="
               color: #000000;">
-            deine E-Mail-Adresse <strong>${email}</strong> wurde benutzt, um ein Wire-Benutzerkonto zu erstellen. Wir möchten gerne sicherstellen, dass du <strong>${name}</strong> bist.
+            <strong>${email}</strong> wurde verwendet, um ein Draht-Konto erstellen. Überprüfe deine e-Mail-Adresse.
```

Comment:
`Draht-Konto`? And _zu_ erstellen.

Reply:
`<strong>${email}</strong> wurde verwendet, um ein Wire Benuzterkonto zu erstellen.` (The `Überprüfe` bit is unnecessary as the next sentence already states the same.